explainaboard/info.py

```diff
@@ -47,23 +40,23 @@ class PaperInfo:
 
 @dataclass
 class Performance:
-    metric_name: str = None
-    value: float = None
+    metric_name: Optional[str] = None
+    value: Optional[float] = None
     confidence_score_low: Optional[float] = None
     confidence_score_high: Optional[float] = None
 
 
 @dataclass
 class BucketPerformance(Performance):
-    bucket_name: str = None
```

Comment:
`bucket_name` and `n_samples` are not actually optional.

Reply:
This is not allowed by [PEP-557](https://peps.python.org/pep-0557/#specification) because:
- The members in `BucketPerformance` follows all the members in `Performance`
- Attributes without defaults (`bucket_name` here) could not follow attributes with defaults (`confidence_score_high` here)

Reply:
Subclassing dataclasses is usually not a good choice because of this kind of problems. In this case owning `Performance` by `BucketPerformance` is the correct way IMO:

```python
@dataclass
def BucketPerformance:
  ...members...
  performance: Performance
```